inspire/modules/migrator/manage.py

```diff
@@ -61,7 +61,9 @@
                 help='Specific collections to migrate.')
 @manager.option('-t', '--input-type', dest='input_type', default='marcxml',
                 help="Format of input file.")
-def populate(records, collections, file_input=None, input_type=None):
+@manager.option('--force', action='store_true', dest='force_import', default=None,
+                help="Force records that are not registered to import on the system")
+def populate(records, collections, file_input=None, input_type=None, force_import=None):
     """Train a set of records from the command line.
 
     Usage: inveniomanage predicter train -r /path/to/json -o model.pickle
```

Comment:
Yeah, this docstring is not correct (my bad). @Panos512  Regarding the usage, those arguments are not applicable for this function. You can see the arguments that belong to this function as listed in the decorators `@manager.option`.